src/plugins/data/public/ui/dataset_selector/configurator.tsx

```diff
@@ -202,7 +305,7 @@ export const Configurator = ({
         </EuiButton>
         <EuiButton
           onClick={async () => {
-            await queryString.getDatasetService().cacheDataset(dataset, services);
+            await queryString?.getDatasetService().cacheDataset(dataset, services);
```

Comment:
queryString manager isn't optional. if this related to a test though we should fix the test to instantiate these properly vs updating the code.